src/contracts/interfaces/IStakeRegistry.sol

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "./IRegistry.sol";
+
+/**
+ * @title Interface for a `Registry` that keeps track of stakes of operators for up to 256 quroums.
+ * @author Layr Labs, Inc.
+ */
+interface IStakeRegistry is IRegistry {
+    // DATA STRUCTURES
+
+    /// @notice struct used to store the stakes of an individual operator or the sum of all operators' stakes, for storage
+    struct OperatorStakeUpdate {
+        // the block number at which the stake amounts were updated and stored
+        uint32 updateBlockNumber;
+        // the block number at which the *next update* occurred.
+        /// @notice This entry has the value **0** until another update takes place.
```

Comment:
just commenting that it seems like there's a good property or two to check here

Reply:
agree